src/Compilers/CSharp/Portable/Lowering/LocalRewriter/DelegateCacheContainer.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols;
+
+internal sealed class DelegateCacheContainer : SynthesizedContainer
```

Comment:
>DelegateCacheContainer [](http://example.com/codeflow?start=22&length=22)

Consider adding a doc comment with a brief description of the purpose for the type.

Reply:
Added. Please take a look.